packages/veui-theme-dls/components/Popover.js

```diff
@@ -0,0 +1,10 @@
+import config from 'veui/managers/config'
+
+config.defaults(
+  {
+    parts: {
+      style: 'reverse'
```

Comment:
```suggestion
      self: 'reverse'
```

这个 `parts` 是一个组件中间的部件名字，怎么能叫 `style` 呢？